src/GraphQL/SortBlockMutationCreator.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace DNADesign\Elemental\GraphQL;
+
+use DNADesign\Elemental\Models\BaseElement;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use SilverStripe\GraphQL\MutationCreator;
+use SilverStripe\GraphQL\OperationResolver;
+use SilverStripe\ORM\DB;
+
+class SortBlockMutationCreator extends MutationCreator implements OperationResolver
+{
+    public function attributes()
+    {
+        return [
+            'name' => 'sortBlock',
+            'description' => 'Changes the sort position of an element'
+        ];
+    }
+
+    public function type()
+    {
+        return $this->manager->getType('Block');
+    }
+
+    public function args()
+    {
+        return [
+            'ID' => ['type' => Type::nonNull(Type::id())],
+            'AfterBlockID' => ['type' => Type::nonNull(Type::id())],
+        ];
+    }
+
+    public function resolve($object, array $args, $context, ResolveInfo $info)
+    {
+        $block = BaseElement::get_by_id($args['ID']);
+
+        if (!$block) {
+            throw new \InvalidArgumentException(sprintf(
+                '%s# not found',
+                ElementArea::class,
+                $args['ID']
+            ));
+        }
+
+        $parentId = $block->ParentID;
+        $blockPosition = $block->Sort;
+
+        $sortAfterPosition = 0;
+        $afterBlockID = $args['AfterBlockID'];
+        if ($afterBlockID) {
+            $afterBlock = BaseElement::get_by_id($afterBlockID);
+
+            if (!$afterBlock) {
+                throw new \InvalidArgumentException(sprintf(
+                    '%s#%s not found',
+                    BaseElement::class,
+                    $afterBlockID
+                ));
+            }
+            if ($afterBlock->ParentID !== $parentId) {
+                throw new \InvalidArgumentException(sprintf(
+                    '%s# not found',
```

Comment:
In this case the message should be something that indicates "Trying to sort block after a block in a different elemental area" or words to that effect.